code/modules/mining/ores_coins.dm

```diff
@@ -333,18 +334,33 @@ GLOBAL_LIST_INIT(sand_recipes, list(\
 /obj/item/coin
 	icon = 'icons/obj/economy.dmi'
 	name = "coin"
-	icon_state = "coin__heads"
+	icon_state = "coin"
 	flags_1 = CONDUCT_1
 	force = 1
 	throwforce = 2
+	custom_materials = list(/datum/material/iron = 400)
+	material_flags = MATERIAL_ADD_PREFIX | MATERIAL_COLOR | MATERIAL_AFFECT_STATISTICS
 	w_class = WEIGHT_CLASS_TINY
 	var/string_attached
 	var/list/sideslist = list("heads","tails")
-	var/cmineral = null
 	var/cooldown = 0
-	var/value = 1
+	var/value
 	var/coinflip
 
+/obj/item/coin/Initialize()
```

Comment:
watch for this aswell, changed how material coins are generated might interfere with unique sprites on coins